### PR TITLE
[Swift 4] Fix not unwrapped optional value

### PR DESCRIPTION
**Pull request by adamkaplan**
Simple fix for an optional string that was not unwrapped prior to using it.

### CODE REVIEW

**Comment by adamkaplan** [comment]
@glock45 wdyt?

File: Sources/HttpParser.swift [hunk 1]
```diff
@@ -37,10 +37,11 @@ public class HttpParser {
             return []
         }
         let queryStart = url.index(after: questionMark)
-        guard url.endIndex > queryStart else {
-            return []
-        }
-        let query = String(url[queryStart..<url.endIndex])
+
+        guard url.endIndex > queryStart,
+            let query = String(url[queryStart..<url.endIndex])
```
Comment (JUSTINMKAUFMAN) [inline Sources/HttpParser.swift]: This already returned a non-optional string. This change shouldn't even compile as far as I can tell.

**Review by JUSTINMKAUFMAN — CHANGES_REQUESTED** [review]

File: Sources/HttpParser.swift (hunk 1 above)
Comment (adamkaplan) [reply Sources/HttpParser.swift]: That works up to Swift 3, it turns out. Swift 4 does enforce conditional binging in guards in my test today...

**Comment by adamkaplan** [comment]
@JUSTINMKAUFMAN Fixed it!

**Pull request merged**